speaker/speaker.go

```diff
@@ -141,13 +140,14 @@ func (s *Speaker) sendOpen(ro *msg.Open) {
 	// FIXME: add real capabilities
 	o.Caps.Use(caps.CAP_EXTENDED_MESSAGE)
 	o.Caps.Use(caps.CAP_ROUTE_REFRESH)
-	if mp, ok := o.Caps.Use(caps.CAP_MP).(*caps.MP); ok {
-		mp.Add(af.AFI_IPV4, af.SAFI_UNICAST)
-		mp.Add(af.AFI_IPV4, af.SAFI_FLOWSPEC)
-
-		mp.Add(af.AFI_IPV6, af.SAFI_UNICAST)
-		mp.Add(af.AFI_IPV6, af.SAFI_FLOWSPEC)
-	}
+	// don't add more MP capablities that we set
```

Comment:
I don't know why this was here
but it my case it was negotiating IPV6 unicast MP, whereas I only wanted to do IPv4 unicast.

I commented it here, but I don't know if you actually want to keep it or not.

The piece of code I was using:
```
mp := caps.NewMP(caps.Code(0))
	mp.(*caps.MP).Add(af.AFI_IPV4, af.SAFI_UNICAST)
	cs := caps.Caps{}
	cs.Set(caps.CAP_MP, mp)
	cs.Set(caps.CAP_AS4, caps.NewAS4(caps.CAP_AS4))
	addPathIpv4 := caps.NewAddPath(0)
	addPathIpv4.(*caps.AddPathCap).Add(af.AFI_IPV4, af.SAFI_UNICAST, caps.AddPathDirection{
		Send:    false,
		Receive: true,
	})

	cs.Set(caps.CAP_ADDPATH, addPathIpv4)
	// R side: a local speaker, sending to L
	spk := speaker.NewSpeaker(context.Background())
	spk.Options.LocalCaps = cs
```